cgmanifest.json

```diff
@@ -6925,13 +6925,23 @@
         }
       }
     },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "ibmtpm",
+          "version": "0.1682",
+          "downloadUrl": "https://sourceforge.net/projects/ibmswtpm2/files/ibmtpm1682.tar.gz"
+        }
+      }
+    },
     {
       "component": {
         "type": "other",
         "other": {
           "name": "ibus",
           "version": "1.5.22",
-          "downloadUrl": "https://github.com/ibus/ibus/releases/download/1.5.22/ibus-1.5.22.tar.gz"
+          "downloadUrl": "https://sourceforge.net/projects/ibmswtpm2/files/ibmtpm1682.tar.gz"
```

Comment:
this doesn't look right?

Reply:
the actual url has `/download` at the end, but otherwise this is correct.

Reply:
you are totally right, sorry.  thanks for repeating :)